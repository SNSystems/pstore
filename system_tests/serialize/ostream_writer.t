# %binaries = the directories containing the executable binaries
# %T = the test output directory
# %S = the test source directory
REQUIRES: examples
RUN: %binaries/example_ostream_writer > %T/ostream_writer_actual.txt
RUN: diff %T/ostream_writer_actual.txt %S/ostream_writer_expected.txt
