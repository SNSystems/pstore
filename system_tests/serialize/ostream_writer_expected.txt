179
127
73
